qemu/tests/pci_devices.py

```diff
@@ -264,7 +264,7 @@ def add_virtio_disk(params, name_idxs, parent_bus, addr):
     params['image_size_%s' % name] = '1M'
     if not params.get('reserved_slots_%s' % parent_bus):
         params['reserved_slots_%s' % parent_bus] = ""
-    params['reserved_slots_%s' % parent_bus] += " %02x-00" % addr
+    params['reserved_slots_%s' % parent_bus] += " 0x%x-0x0" % addr
```

Comment:
I found another similar format string in line 53:

```python
            if len(addr) == 1:
                addr.append(0)
            addr = "%02x.%x" % (addr[0], addr[1])
            dev = {'id': dev_id,
                   'type': device.get_param('driver'),
                   'bus': device.get_param('bus'),
                   'addr': addr}
```

Reply:
Yep, I noticed the failures today while processing results. I'll send v2 with that fixed as well.

Reply:
Actually not, that is a different issue. What is the problem in here? I think it's fine as the qtree and lspci uses hex numbers.